recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "nnpops" %}
+{% set version = "0.1" %}
```

Comment:
Tomorrow I can make the release of 0.2. So, you are building the latest code.

Reply:
Sounds good, can you ping me on slack or on github? Then I will update this recipe to use the latest release.